packages/material-ui/src/CardHeader/CardHeader.js

```diff
@@ -56,7 +56,7 @@ function CardHeader(props) {
   if (title != null && title.type !== Typography && !disableTypography) {
     title = (
       <Typography
-        variant={avatar ? 'body2' : 'headline'}
+        variant={avatar ? 'subheading' : 'headline'}
```

Comment:
Should add a note to the release notes that CardHeader uses a different variant.

Reply:
This change is just to better match the list specification that changed in v2. It's unrelated to the other changes. We have an open issue regarding matching the spec. It's one step closer to this target.

Reply:
I agree with the change and I'm totally fine with it. But better add one warning to many instead of a silent change.